src/MCT.js

```diff
@@ -73,37 +72,46 @@ import Browse from './ui/router/Browse.js';
  * The Open MCT application. This may be configured by installing plugins
  * or registering extensions before the application is started.
  * @constructor
- * @memberof module:openmct
- * @extends EventEmitter
  */
 export class MCT extends EventEmitter {
+  /**
+   * @type {import('openmct.js').BuildInfo}
+   */
+  buildInfo;
+  /**
+   * @type {string}
+   */
+  defaultClock;
+  /**
+   * @type {Record<string, OpenMCTPlugin>}
+   */
+  plugins;
+  /**
+   * Tracks current selection state of the application.
+   * @type {Selection}
+   */
+  selection;
   constructor() {
     super();
 
     this.buildInfo = {
+      // @ts-ignore
```

Comment:
is there a way we can add this to our ts config in some fashion? like we aded them to the .eslintrc?

Reply:
ooooo i think these actually snuck in when i was doing some experimenting. we have ts-check disabled, but i enabled it for a while to guide my type updates